packages/core/src/types.ts

```diff
@@ -89,7 +88,7 @@ export interface Location {
   radius?: number
 }
 
-type GuestsRatingBreakdown =
+export type GuestsRatingBreakdown =
```

Comment:
```suggestion
export type GuestRatingBreakdown =
```
Since you call it GuestRating later, and I think it's the better name.